.github/workflows/ci-cd-pims-dev.yml

```diff
@@ -96,24 +98,87 @@ jobs:
           openshift_token: ${{ env.OPENSHIFT_TOKEN }}
           insecure_skip_tls_verify: true
           namespace: ${{ env.OPENSHIFT_TOOLS_NAMESPACE }}
+      - name: Pre-deploy scan frontend (block on CRITICAL/HIGH)
+        env:
+          IMAGE: ${{ env.OPENSHIFT_REGISTRY }}/3cd915-tools/pims-app:latest
+        run: |
+          docker pull "$IMAGE";
+          trivy image --exit-code 1 --scanners vuln,secret,misconfig --format table --severity CRITICAL,HIGH --ignore-unfixed "$IMAGE" | tee frontend_predeploy_scan.txt;
```

Comment:
Are we sure we want the --ignore-unfixed flag? it might be nice to know that a critical, unpatched vulnerability exists.

Reply:
this flag works in the opposite way:
"This flag excludes vulnerabilities for which there are no fixed versions"
https://github.com/aquasecurity/trivy/discussions/1687

Do we want to include these vulnerabilities?